staking/types/staking_transaction.go

```diff
@@ -0,0 +1,26 @@
+package types
+
+import (
+	"math/big"
+
+	"github.com/harmony-one/harmony/internal/common"
+)
+
+// StakingTransaction struct.
+type StakingTransaction struct {
+	AccountNonce uint64         `json:"nonce"      gencodec:"required"`
```

Comment:
renamed to Nonce ?

Reply:
Just following the transaction field. and This is the same as tx's nonce.